server/repositories/TransferRepository.js

```diff
@@ -28,6 +28,18 @@ class TransferRepository extends BaseRepository{
       state: Transfer.STATE.pending,
     });
   }
+
+  async getTokensById(id){
```

Comment:
Renaming this to `getTokenAndCaptureIds(transactionId)` would be verbose but more clear. Right now, it is not clear the parameter  `id` is referring the transaction/transfer id or token id.

Another lesser preference would be to rename it to `getTokens(transactionId)`

Reply:
Good, I agree, I will change it